src/cfnlint/helpers.py

```diff
@@ -166,10 +167,63 @@
     'AWS::Redshift::Cluster'
 ]
 
+CACHING_DIR = os.path.join(os.path.dirname(__file__), 'data', 'DownloadsMetadata')
 
-def get_url_content(url):
+
+def get_url_content(url, caching=False):
     """Get the contents of a spec file"""
```

Comment:
To be more specific, the function returns `None` if the spec file is in the cache.